ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh

```diff
@@ -140,7 +140,7 @@ if [[ "${SIZE_VARIANT}" == "compact" ]]; then
   echo "Compact SIZE_VARIANT was configured, setting worker's replicas to 0"
   MACHINE_POOL_OVERRIDES="controlPlane:
   name: master
-  replicas: 3
+  replicas: ${CONTROL_PLANE_REPLICAS}
```

Comment:
For compact one, we only treat 3 masters as a compact cluster, so we can use hard-code - `3` here.

Reply:
ok

Reply:
seems this is not compact scenario, see installation doc: [https://docs.openshift.com/container-platform/4.10/installing/installing_vsphere/installing-vsphere.html#installation-vsphere-config-yaml_installing-vsphere](url)  the worker replicas in upi install-config is default 0.